scripts/concurrent_task_utils_test.py

```diff
@@ -22,14 +22,13 @@
 import threading
 import time
 
-from core import python_utils
 from core.tests import test_utils
 
 from . import concurrent_task_utils
 
 
 def test_function(unused_arg):
-    return python_utils.OBJECT
+    return builtins.object
```

Comment:
Why `builtins.object` instead of just `object`? It looks like they're the same:

```python
>>> import builtins
>>> builtins.object is object
True
```

Reply:
yes it is the same, I kept it like that just becuse of the original implementation in python_utils, but I guess that was done to support python2
So I have kept it as just `object` PTAL